code/game/machinery/_machinery.dm

```diff
@@ -345,7 +346,12 @@
 		if(!(interaction_flags_machine & INTERACT_MACHINE_ALLOW_SILICON))
 			return FALSE
 
-	else if(isliving(user)) // If we are a living human
+	if(istype(user, /mob/living/simple_animal))
+		var/mob/living/simple_animal/SA = user
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Change this to specify this is the user cast as a simple animal.

Reply:
hope `user_as_animal` is good enough

Reply:
Yeah.